docs/setup-guide/scale-out-local-dev-setup.md

```diff
@@ -34,23 +34,40 @@ export RESOURCE_PARTITION_IP=[RP_IP]
 1. Set up environment variables
 
 ```
-export SCALE_OUT_PROXY_ENDPOINT=http://[PROXY_IP]:8888
+export SCALE_OUT_PROXY_IP=[PROXY_IP]
+export SCALE_OUT_PROXY_PORT=8888
 export IS_RESOURCE_PARTITION=false
+export RESOURCE_SERVER=[RP_IP]
+export REUSE_CERTS=true
 ```
 
 1. Run ./hack/arktos-up-scale-out-poc.sh
 
 1. Expected last line of output: "Tenant Partition Cluster is Running ..."
 
+Note:
+
+1. As we start to picking up secure mode in scale out, api server certificates will be shared across all partitions in 
+development environment. The first TP that started needs to generate api server certificates and be copied over to other
+TP/RP before they start.
+
+1. To generate first set of certs, run `REUSE_CERTS=false; ./hack/arktos-up-scale-out-poc.sh`
```

Comment:
the IPs ( or master name ) for the other cluster's master will needed to set the proper SANs or the certs.

Reply:
This is user manual. It's better let the tool handle the terminology. I would avoid add technical terms here.